tslint.json

```diff
@@ -25,6 +26,7 @@
     "no-console": [false],
     "no-empty-interface": false,
     "no-string-literal": false,
+    "no-unused-expression-chai": false,
```

Comment:
Doesn't this disable the rule?

Reply:
Nice catch. While revisiting this I realized that I was actually disabling the `no-unused-expression` rule for every kind of expression.
Removed this and used `const { expect } = m.chai;` to make tslint detect that it's a chai expression.